Library/Homebrew/env_config.rb

```diff
@@ -19,9 +19,10 @@ module EnvConfig
         description: "Automatically check for updates once per this seconds interval.",
         default:     300,
       },
-      HOMEBREW_BAT:                       {
-        description: "If set, use `bat` for the `brew cat` command.",
-        boolean:     true,
+      HOMEBREW_BAT_CONFIG_PATH:           {
+        description:  "If set, use `bat` for the `brew cat` command. This can be the path to the `bat` " \
+                      "configuration file. For example, `HOMEBREW_BAT=$XDG_CACHE_HOME/bat/config`.",
```

Comment:
```suggestion
                      "configuration file. For example, `HOMEBREW_BAT=$HOME/.bat/config`.",
```

Reply:
Actually this should probably just match `bat --config-file`, or the default `~/.config/bat/config` if no `XDG_*` are set…

Reply:
I think having it be non-default is more obvious that users don't need to set it to have it picked up unless it deviates from the default.

Reply:
> users don't need to set it to have it picked up unless it deviates from the default

Well blocking `XDG_*` means they do have to bother with it currently… but I see your point. Sorted now.